cetmix_tower_server/tests/test_server_template.py

```diff
@@ -295,3 +295,348 @@ def test_server_template_copy(self):
             original_log.command_id.code,
             "Command code should be the same in the copied server log",
         )
+
+    def test_required_flag_in_server_template_configuration(self):
+        """
+        Test that the 'Required' checkbox is displayed and saved correctly
+        in the Configuration tab of the Server Template.
+        """
+        # Create a variable with the required flag
+        self.VariableValue.create(
+            {
+                "variable_id": self.variable_version.id,
+                "server_template_id": self.server_template_sample.id,
+                "value_char": "Test Value",
+                "required": True,
+            }
+        )
+
+        # Checking that the variable is saved with the required flag
+        variable = self.VariableValue.search(
```

Comment:
This is an ORM function which we are not modifying. So no need to test its behaviour because this is done by the ORM tests.